rest/servers/spring/src/main/java/com/connexta/ingest/rest/spring/IngestApi.java

```diff
@@ -72,30 +72,32 @@
   default ResponseEntity<Void> ingest(
       @ApiParam(
               value =
-                  "The minimal API version that a client using this API will accept responses from. ",
+                  "The minimal API version from which the client using this API will accept responses. ",
               required = true)
           @RequestHeader(value = "Accept-Version", required = true)
           String acceptVersion,
       @ApiParam(
               value =
-                  "The last modified time of the product to be ingested. ",
+                  "The last modified time of the dataset to be ingested. ",
               required = true)
           @RequestHeader(value = "Last-Modified", required = true)
               OffsetDateTime lastModified,
       @ApiParam(
-              "A file attachment that is sent in the request. The current maximum is the character equivalent of 10GB.")
+              value =
+                  "A file attachment that is sent in the ingest request. The current maximum is the character equivalent of 10GB.",
+              required = true)
```

Comment:
I noticed that this API param was not in the same format as the same annotation for the metacard, so I added the `value` and `required` pieces.